.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    branches: # Snapshots
+      - main
+    tags: ["*"] # Releases
+
+jobs:
+  publish-artifacts:
+    name: Publish / Artifacts
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v13
+        with:
+          java-version: adopt@1.8
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.2
+
+      - name: Publish artifcats
+        run: sbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
I will need to add the secrets. They are not there yet. 
We may want to wait a bit with that though. @mkurz is getting a new sonatype user that we will use to publish artifacts. 

Once we have the user, we can add the secrets at organisation level. So far we have done per repo, but maybe it's better to do once and for all at org level. Anyway, to follow-up...

Reply:
We can simply set it up once for the org, done. I'll do that as soon I have the go from sonatype staff.

Reply:
The secrets are set up now. Happy publishing.